components/elm/src/biogeochem/CNPhenologyBeTRMod.F90

```diff
@@ -1893,13 +1894,17 @@ subroutine CropPhenologyInit(bounds)
     ! Convert planting dates into julian day
     minplantjday(:,:) = huge(1)
     maxplantjday(:,:) = huge(1)
-    do n = npcropmin, npcropmax
```

Comment:
I see, so this is an example of added flexibility to where cfts are positioned in the list, we no longer rely on looping through the expected cft bounds, but go through all fts and check their iscft label...

Reply:
Is it possible to remove the usage of npcropmin and npcropmax altogether, and remove it from pftvarcon?

Reply:
That is something we can try to tackle in later PRs, focusing on code cleanup.